Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rusk"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
I don't see any point in starting to version this when it's not ready to be used at all. Dunno the opinion of the others.

Reply:
Fixed